source/tests/pt/test_make_stat_input.py

```
@@ -0,0 +1,89 @@
+        lst = make_stat_input(
+            datasets=self.datasets,
+            dataloaders=self.dataloaders,
+            nbatches=nbatches,
+            min_frames_per_element_forstat=1,
+            enable_element_completion=True,
+        )
+        coll_ele = set()
+        for i in lst:
+            ele = np.unique(i["atype"].cpu().numpy())
+            coll_ele.update(ele)
+        if not coll_ele == self.alltype:
```

Ruff (0.8.2)

Use `coll_ele != self.alltype` instead of `not coll_ele == self.alltype`

Replace with `!=` operator

(SIM201)